Lib/test/test_pkgutil.py

```diff
@@ -522,6 +522,54 @@ def test_mixed_namespace(self):
         del sys.modules['foo.bar']
         del sys.modules['foo.baz']
 
+
```

Comment:
Only one blank line between methods, per [PEP 8](https://peps.python.org/pep-0008/)
```suggestion
```

Reply:
Removed the blank line. Also, I've learned that you prefer not to have force-pushes on CPython PRs. I'll respect that for future PRs. :)